README.md

```diff
@@ -1,4 +1,4 @@
-# StationBench
+# StationBench v0.1.0
```

Comment:
lets not do this in the code because when I make this a release we have to update this all the time, and if we forget it will diverge